cogs/draw_image_generation.py

```diff
@@ -6,20 +6,19 @@
 
 import discord
 from PIL import Image
-from discord.ext import commands
+from pycord.multicog import add_to_group

```

Comment:
What is `pycord`? I thought the package name was `py-cord`, or referenced as `discord`, `pycord` doesn't seem to exist?

![image](https://user-images.githubusercontent.com/21161101/210284339-dc4bf3a5-a67b-4bb9-8dcd-45461d0e46e0.png)

Reply:
I'm assuming this is the package? 
![image](https://user-images.githubusercontent.com/21161101/210284412-9f98c738-732a-48ed-a5f3-b684d15d2ddf.png)

Reply:
Is there any way to achieve what you've done here without using another package?

Reply:
Haha, it's the name of the extension, see this https://github.com/Dorukyum/pycord-multicog you probably have to install requirements again since I added it there